README.md

```diff
@@ -237,6 +279,18 @@ etc.). This can be done as follows:
   to the table's list of columns.
 
 
+#### Missing data
+
+Data creators often want to distinguish two kinds of missing data, in particular when the data is extracted from
+sources:
+1. data that is missing/unknown because it was never extracted from the source,
+2. data that is indicated in the source as unknown.
+
+The CSVW data model can be used to express this difference as follows:
+Case 1 can be modeled by not including the relevant data as row at all.
+Case 2 can be modeled using the `null` property of the relevant column specification as value in a data row.
```

Comment:
is there a default value for case 2?

Reply:
Yes, empty string, see https://github.com/cldf/csvw/blob/179fc59e460de924a149687ae3566471fa8fb431/src/csvw/metadata.py#L352-L354